src/OpenTelemetry.Instrumentation.EntityFrameworkCore/OpenTelemetry.Instrumentation.EntityFrameworkCore.csproj

```diff
@@ -1,17 +1,24 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Microsoft.EntityFrameworkCore instrumentation for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
     <MinVerTagPrefix>Instrumentation.EntityFrameworkCore-</MinVerTagPrefix>
-    <IncludeSharedInstrumentationSource>true</IncludeSharedInstrumentationSource>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Api" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsPkgVer)" />
+    <PackageReference Include="OpenTelemetry.Api.ProviderBuilderExtensions" Version="$(OpenTelemetryCoreLatestPrereleaseVersion)" />
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Contrib.Shared\Api\ExceptionExtensions.cs" Link="Includes\ExceptionExtensions.cs"/>
```

Comment:
Separate PR:
Exclude from `IncludeSharedInstrumentationSource` files which depends on DynamicMethod.
Crate separate Property to include onlye DynamibMethod based clases do this, revert changes in this file.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1028

Reply:
@Kielek I didn't notice you create https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1026, should I close the https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1027?

Reply:
You have linked PR to your issue. I have closed #1026